proof_ipa.go

```diff
@@ -282,7 +282,7 @@ type stemInfo struct {
 }
 
 // TreeFromProof builds a stateless tree from the proof
-func TreeFromProof(proof *Proof, rootC *Point) (VerkleNode, error) {
+func TreeFromProof(proof *Proof, rootC *Point) (VerkleNode, error) { // skipcq: GO-R1005
```

Comment:
@gballet, [DeepSource thought it was a good idea today ](https://app.deepsource.com/gh/gballet/go-verkle/run/b42ab770-414f-49ec-a49c-48de24cae7f7/go/)to change their criteria and now was complaining about the cyclomatic complexity of this function since it consider is too complex. 

May be right, but right now you're around these zones and might not be a good idea to try to solve that in this PR (which is unrelated).

Reply:
yeah saw that, thank you. Ideally we would fix it altogether but let's skip that for now.